docs/openfaas-pro/introduction.md

```diff
@@ -58,6 +58,63 @@ Build functions at scale - for services providers and large teams:
 
 * [Build functions via REST API](/openfaas-pro/builder) using source code without the need to create and manage dozens or hundreds of independent CI jobs.
 
+### Grafana dashboards
```

Comment:
I was thinking these would be broken out into separate sections in the ToC

Reply:
Created a sub section for each dashboard.